debian/control

```diff
@@ -0,0 +1,16 @@
+Source: dangerzone
+Maintainer: Freedom of the Press Foundation <info@freedom.press>
+Section: python
+Priority: optional
+Build-Depends: dh-python, python3-setuptools, python3-all, debhelper (>= 9), dh-python, python3, python3-setuptools
```

Comment:
* dh-python and python3-setuptools are listed twice
* you don't need both python3-all and python3, just the former is fine